internal/orchestrator/updates.go

```diff
@@ -94,25 +94,20 @@ func (o *Orchestrator) kvStatusPublisher(ctx context.Context) {
 	o.logger.Debug("shut down KV updates")
 }
 
-type translatorFn func(context.Context, nats.KeyValueEntry) (*v1types.ConditionUpdateEvent, error)
-
 // startConditionWatchers does what it says on the tin; iterate across all configured conditions
 // and start a KV watcher for each one. We increment the waitgroup counter for each condition we
 // can handle.
 func (o *Orchestrator) startConditionWatchers(ctx context.Context,
 	evtChan chan<- *v1types.ConditionUpdateEvent, wg *sync.WaitGroup) {
 
 	for _, def := range o.conditionDefs {
-		var tf translatorFn
 		var watcher nats.KeyWatcher
 
 		kind := def.Kind
 
 		switch kind {
```

Comment:
Can we drop this check as well, so its not limiting if we add new conditions or rename an existing one.

Reply:
As discussed, I'll remove this check.